test/test-lint.ts

```diff
@@ -66,6 +71,37 @@ test.serial('lint should return false on bad code', async t => {
       });
 });
 
+test.serial('lint should auto fix fixable errors', async t => {
+  await withFixtures(
+      {
+        'tsconfig.json': JSON.stringify({files: ['a.ts']}),
+        'a.ts': FIXABLE_CODE
+      },
+      async (fixturesDir) => {
+        const okay = lint.lint(OPTIONS, [], true);
+        t.is(okay, true);
+        const contents =
+            fs.readFileSync(path.join(fixturesDir, 'a.ts'), 'utf8');
+        t.deepEqual(contents, FIXABLE_CODE_FIXED);
+      });
+});
+
+test.serial('lint should auto fix fixable errors unless dry-run', async t => {
```

Comment:
The description sounds a little bit off from what is tested. Maybe 'lint should not auto fix on dry-run'?

Reply:
done.